examples/hacker-news-analyzer/tsconfig.json

```
@@ -10,11 +11,14 @@
     "moduleResolution": "NodeNext",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "react-jsx",
-    "jsxImportSource": "@gensx/core",
     "rootDir": ".",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "plugins": [
+      {
+        "name": "ts-function-decorator-ls"
+      }
+    ]
   },
-  "include": ["./src/**/*.ts", "./src/**/*.tsx"],
-  "exclude": ["node_modules"]
+  "include": ["./*.ts", "./*.tsx", "./**/*.ts", "./**/*.tsx"],
```

I assume we no longer need tsx?
```suggestion
  "include": ["./*.ts", "./**/*.ts"],
```